bigquery-writer/bigquery-writer.py

```diff
@@ -97,38 +160,37 @@ def start(self):
             sleep(1000)
             return
```

Comment:
Can all the checks above be consolidated in one function? They all call `sleep` and return.